api/serverpb/server.proto

```diff
@@ -232,6 +234,10 @@ message ChangeSettingsRequest {
   bool remove_pmm_public_address = 20;
   // Intervals between STT check runs.
   STTCheckIntervals stt_check_intervals = 21;
+  // Enable Azure Discover.
+  bool enable_azurediscover = 22;
+  // Disable Azure Discover.
+  bool disable_azurediscover = 23;
```

Comment:
Maybe disable_azure_discover?

Reply:
IMHO we are not going to have extended azure variables. So it would require to distinguish the third part.
e.g.
disable_azure_parameter1
disable_azure_parameter2